### PR TITLE
feat: add multiarch builds

### PR DESCRIPTION
**Pull request by mhaley-tignis**

### CODE REVIEW

**Review by silviudruma — APPROVED** [review]
Great action. It will let us run the images on Linux or Mac

File: action.yml [hunk 1]
```diff
@@ -74,6 +78,7 @@ runs:
         push: ${{ inputs.push }}
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+        platforms: ${{inputs.platforms}}
```
Comment (steven-herchak) [inline action.yml:81]: Should we distribute the builds for different platforms?
https://docs.docker.com/build/ci/github-actions/multi-platform/

Comment (steven-herchak) [reply action.yml:81]: This would use multiple runners (effectively run parallel insteaed of in series) resulting in lower build time.

Comment (mhaley-tignis) [reply action.yml:81]: The caller of this action can setup parallel runners if they desire.

**Pull request merged**